utils/install.py

```diff
@@ -610,6 +615,27 @@ def fix_gnu_sparse(args):
                     shutil.rmtree(join(args.path, dir, sub_dir))
 
 
+def check_nvcc(platform):
+    if platform == "Linux":
+        cmd = "/usr/local/cuda/bin/nvcc --version"
+        try:
+            output = subprocess.check_output([cmd], shell=True).decode("utf8").strip()
+        except subprocess.CalledProcessError as e:
+            logging.debug(
+                "Exception on checking nvcc - rc= %s output= %s command= %s",
+                e.returncode,
+                e.output,
+                e.cmd,
+            )
+            return False
+        logging.debug("%s: %s", cmd, output)
+        if "nvcc: NVIDIA (R) Cuda compiler driver" in output:
+            return True
+    else:
+        logging.warning("CUDA should be only available on Linux")
+    return False
```

Comment:
Can you try to use https://github.com/WasmEdge/WasmEdge/blob/5cfcabb6ab3a1009e15f92244ea30f506d1c080f/utils/install.py#L1256 ?

Reply:
Because `run_shell_command` will print an error log like the following section:

```
ERROR:root:Exception on process - rc= 127 output=  command= [u'/usr/local/cuda/bin/nvcc --version']
False
```

However, the only thing I want to do is to detect if the environment has cuda or not. Hence, using this function will make users confused.

Reply:
`/usr/local/cuda/bin/nvcc --version 2>/dev/null` if empty then it does not exist.

Also should we check for /usr/local/cuda/bin path only or is it possible for it to be in other paths?

Reply:
Alrighty, thanks!

Reply:
> `/usr/local/cuda/bin/nvcc --version 2>/dev/null` if empty then it does not exist.
> 

Nice catch! Will process this.

Reply:
No luck. The issue persists.

```
ERROR:root:Exception on process - rc= 127 output=  command= [u'/usr/local/cuda/bin/nvcc --version 2>/dev/null']
False
```

Reply:
Since this is a minor issue, I will use `run_shell_command` instead from now.